node_modules/oae-principals/emailTemplates/verify.html.jst

```diff
@@ -0,0 +1,195 @@
+             *************/
+
+            body {
+                background-color: <%= skinVariables['body-background-color'] %>;
+                color: <%= skinVariables['text-color'] %>;
+                font-size: 13px;
+                height: 100%;
+                padding: 20px;
+                -webkit-font-smoothing: antialiased;
+                -webkit-text-size-adjust: 150%;
+            }
+
+            body, h1, h2, h3, h4, h5, h6, a, span, td, div {
+                font-family: HelveticaNeue-Light, "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+            }
+
+            body > table {
+                margin: 0 auto;
+            }
+
+            a {
+                color: <%= skinVariables['link-color'] %>;
+                text-decoration: none;
+            }
+
+            a:hover {
+                color: <%= skinVariables['link-hover-color'] %>;
+            }
+
+            h1, h2, h3, h4, h5, h6 {
+                color: <%= skinVariables['text-color']%>;
+                line-height: 1.1;
+            }
+
+            h1, h2 {
+                font-weight: 200;
+            }
+
+            h3 {
+                font-weight: 500;
+            }
+
+            img {
+                border: 0;
+            }
+
+            .muted {
+                color: <%= skinVariables['secondary-text-color'] %>;
+                font-size: 12px;
+            }
+
+            .wrapped {
+                word-wrap: break-word;
+            }
+
+            .row > td {
+                max-width: 700px;
+                min-width:400px;
+            }
+
+
+            /************
+             ** HEADER **
+             ************/
+
+            #header table {
+                width: 100%;
+            }
+
+            #header td {
+                text-align: center;
+                max-width: 650px;
+            }
+
+            #header #logo img {
+                margin: 0 0 30px;
+
+                /* GMail always converts `height` to `min-height` on at least images, so use
+                   `min-height` with `max-height` for some client consistency */
+                max-height: 50px;
+                min-height: 50px;
+            }
+
+            #header #greeting h1 {
+                font-size: 24px;
+                font-weight: bold;
+                margin: 0 0 5px;
+                word-wrap: break-word;
+            }
+
+            #header #summary td {
+                font-size: 15px;
+                padding-bottom: 10px;
+            }
+
+
+            /*************
+             ** CONTENT **
+             *************/
+
+            .content td {
+                background-color: #FFF;
+                padding: 20px;
+            }
+
+
+            /************
+             ** FOOTER **
+             ************/
+
+            #footer {
+                font-size: 10px;
+                margin: 40px auto 0;
+            }
+
+        </style>
```

Comment:
If there are logical parts (e.g., "header") that are just straight copy-paste with no tuning, then I'd prefer we consolidate it now. If there are minor changes to deal with peculiarities in the design of the email templates, then I don't mind if we out-scope consolidation for more unified design.